### PR TITLE
use the right stringification for wide strings

### PR DESCRIPTION
**Pull request by anporumb**

### CODE REVIEW

File: inc/testrunnerswitcher.h [hunk 1]
```diff
@@ -223,7 +223,7 @@ namespace Microsoft \
             {  \
                 if((q<0)||(size_t)q>=sizeof(EnumName##_Strings)/sizeof(EnumName##_Strings[0])) \
                 { \
-                    return(L"out of range value for " L#EnumName); \
+                    return(L"out of range value for " L"" #EnumName); \
```
Comment (dcristoloveanu) [inline inc/testrunnerswitcher.h:226]: >L"" [](start = 54, length = 3)

Couldn't it be just without this extra L"" then?

**Review by dcristoloveanu — APPROVED** [review]
:shipit:

File: inc/testrunnerswitcher.h (hunk 1 above)
Comment (anporumb) [reply inc/testrunnerswitcher.h:226]: I would assume it could. let me check.

Comment (anporumb) [reply inc/testrunnerswitcher.h]: Is equally good.

**Pull request merged**